blocks/header/header.css

```diff
@@ -1,275 +1,556 @@
 /* header and nav layout */
```

Comment:
I found a few responsive issues

window width: 1000px

![Screenshot 2023-07-12 at 16 52 19](https://github.com/hlxsites/bitdefender/assets/143527/5c3e6ba1-ff4a-4dad-8bd5-c3ab317f3d62)

when I clicked login and then changed window size

![Screenshot 2023-07-12 at 16 55 56](https://github.com/hlxsites/bitdefender/assets/143527/4e19ce07-17ba-419f-9375-4b4feff10982)

when I change window size to <1000px

![Screenshot 2023-07-12 at 16 49 19](https://github.com/hlxsites/bitdefender/assets/143527/bdf36475-fd33-4210-a000-d89c1e2d8f69)

when I open mega-menu on small window size. looks different to bitdefender.com.au

![Screenshot 2023-07-12 at 16 51 46](https://github.com/hlxsites/bitdefender/assets/143527/ec0fa773-6142-4d2f-b28d-b81238d79de0)